src/Core/Tokens/SsoRedirectToken.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+
+namespace Bit.Core.Tokens
+{
+    public static class SsoRedirectToken
+    {
+        #region Constants
+
+        public const string SSO_REDIRECT_COOKIE_NAME    = "SsoRedirect";
+        private const int MAX_REDIRECT_COOKIE_MINUTES   = 5;
```

Comment:
I don't think this is how we name constants. Following the conventions from MS for .NET itself, we use PascalCase.